lms/djangoapps/certificates/management/commands/rescind_cert.py

```diff
@@ -0,0 +1,104 @@
+"""
+Management command to update the status of a single user certificate
+for a given user, in a given course. If no --status kwarg is provided,
+the status will be updated to CertificiateStatuses.unavailable.
+"""
+import logging
+from optparse import make_option
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+
+from certificates.models import GeneratedCertificate, CertificateStatuses
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    '''Update the status of a single user certificate for a given user in a given course'''
+    help = u'Rescind a certificate for a particular user in a particular course.'
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '-c', '--course',
+            metavar='COURSE_ID',
+            dest='course',
+            default=False,
+            help='The course id (e.g., mit/6-002x/circuits-and-electronics) of the course in which the '
```

Comment:
make sure it works with new funky courses?

also, maybe put an example course here so people know what format those course strings should look like...